google-cloud-spanner/src/main/java/com/google/cloud/spanner/MetricRegistryConstants.java

```diff
@@ -45,9 +45,13 @@
   static final String MAX_IN_USE_SESSIONS = "cloud.google.com/java/spanner/max_in_use_session";
   static final String MAX_ALLOWED_SESSIONS = "cloud.google.com/java/spanner/max_allowed_sessions";
   static final String IN_USE_SESSIONS = "cloud.google.com/java/spanner/in_use_sessions";
+  static final String GET_SESSION_TIMEOUT = "cloud.google.com/java/spanner/get_sessions_timeout";
```

Comment:
not sure about the name `get_sessions_timeout`. Maybe `session_open_retries`? This would go well with other metrics `session_opened` and `sessions_closed` if those are implemented.

Reply:
@skuruppu @olavloite any thoughts?

Reply:
No, this retry is not for opening a session, but for getting (or acquiring) a session from the pool. The maximum number of sessions in the pool has already been reached, and the pool is therefore prohibited from opening any new sessions. Instead, the thread that requested the session is placed in a wait queue where it waits until a session is released into the pool by another thread. The waiter will wake up and increase this counter every now and then in order to be able to log information about the fact that it is still waiting.

Reply:
get_session_timeout still doesn't feel right but I don't have any good suggestion. I think it should be  GET_SESSION_TIMEOUTS though ('s at the end).